Added content by adding objective in home page as per issue #146

coding-geek21 opened the pull request:
## Related Issue or bug
  - Adding content to home page

Fixes: #146  

#### Describe the changes you've made
1) Objective of the website is added in the home.html page . It is added below the gif and above the carousel 
2) Added the styles in  <styles> tag 
3) If there is any change or any additional things to be added , you can let me know


## Type of change

Please delete options that are not relevant.
<!--
Example how to mark a checkbox :-
- [x] My code follows the code style of this project.
-->
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. 

#### Describe if there is any unusual behaviour of your code(Write `NA` if there isn't)
A clear and concise description of it.

## Checklist:
<!--
Example how to mark a checkbox :-
- [x] My code follows the code style of this project.
-->
- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] Any dependent changes have been merged and published in downstream modules.

## Screenshots

 Original           | Updated
 :--------------------: |:--------------------:
 **original screenshot**  | <b>updated screenshot </b> |
 
 
![Screenshot 2021-03-14 113205](https://user-images.githubusercontent.com/53329034/111059240-f4742280-84b9-11eb-8992-e72948f45db0.jpg)

sakshikhachane commented:
Hey @coding-geek21,
Please resolve the conflicts.

sakshikhachane commented on main_app/templates/main_app/home.html:
```diff
@@ -183,6 +190,14 @@ <h4 class="User"> &nbsp;&nbsp; {{user.username}} </h4>
 
+  <div class="objectives" style="text-align: center;padding:60px;">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Akaya+Telivigala&display=swap" rel="stylesheet">
```
Please change the font style.

coding-geek21 replied:
Ok @sakshikhachane , I will do the requested changes

coding-geek21 commented:
![image](https://user-images.githubusercontent.com/53329034/111297545-bae71700-8673-11eb-9187-0790346b4094.png)

Changed the font-style of objectives p tag

sakshikhachane commented on main_app/templates/main_app/home.html:
```diff
@@ -106,10 +113,17 @@
         border-left: 0.3em solid transparent;
       }
 
+  <div class="objectives" style="text-align: center;padding:60px;">
```
- [ ] Kindly close the `}` and add this code below `gif image`. The HTML code can't be put into the `styles` tag. 
- [ ] Don't remove the font style. Just replace it with some other good font style that is readable.

sakshikhachane reviewed (changes requested):
@coding-geek21 Please do the necessary changes.

coding-geek21 commented:
Sorry for the mistakes as this was my first time to open source bear with me , i just put the html tag into styles tag by unknowingly by mistake due to confusions 

I have changed the font-style and replaced the objectives correctly below gif 

![image](https://user-images.githubusercontent.com/53329034/111325247-841ff980-8691-11eb-9553-fabcd76b5de7.png)

sakshikhachane commented:
> Sorry for the mistakes as this was my first time to open source bear with me , i just put the html tag into styles tag by unknowingly by mistake due to confusions
> 
> I have changed the font-style and replaced the objectives correctly below gif
> 
> ![image](https://user-images.githubusercontent.com/53329034/111325247-841ff980-8691-11eb-9553-fabcd76b5de7.png)

No problem. Everyone here is student and we are here to learn only :)
